docs/en/serverless/limitations.asciidoc

```diff
@@ -1,5 +1,5 @@
-[[observability-technical-preview-limitations]]
-= Technical preview limitations
+[[observability-limitations]]
```

Comment:
Should we create an AsciiDoc redirect for this in case there are links out in the wild? I could see this as a page that support links to...